Makefile

```diff
@@ -182,9 +182,20 @@ ifndef PCRE_LDFLAGS
  endif
 endif
 
+MIGRATIONS ?= nim_status/lib/migrations/sql_scripts.nim
+
+$(MIGRATIONS): | deps
+	$(ENV_SCRIPT) nim c -r $(NIM_PARAMS) \
+		nim_status/lib/migrations/sql_generate.nim > nim_status/lib/migrations/sql_scripts.nim
+
+clean-migration-file:
+	rm -f nim_status/lib/migrations/sql_scripts.nim
+
+migrations: clean-migration-file $(MIGRATIONS)
```

Comment:
When introducing phony targets they should be added to the `PHONY` list at the top of the Makefile. They will work without doing that but it's best practice and avoids a situation where a file being named the same thing as a phony target causes `make` to behave unexpectedly.